app/Model/helpdesk/Ticket/Ticket_ThreadOld.php

```
@@ -2,43 +2,107 @@
+    |--------------------------------------------------------------------------
+    */
 
-    public function getTitleAttribute($value)
+    public function setTitleAttribute($value)
     {
-        return str_replace('"', "'", $value);
+        if ($value == '') {
+            $this->attributes['title'] = 'No available';
+        } else {
+            $this->attributes['title'] = $value;
+        }
```

PHPMD (2.15.0)

The method setTitleAttribute uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)